app/components/layout/reveal_component/reveal_component_controller.js

```diff
@@ -8,6 +8,19 @@ export default class extends Controller{
 
     static targets = ["hideButton", "showButton", 'item' ]
 
+    connect(){
+        let element
+        if (
+            this.element.childNodes[0] &&
+            this.element.childNodes[0].childNodes[0] &&
+            this.element.childNodes[0].childNodes[0].childNodes[3]
+          ){
+            element = this.element.childNodes[0].childNodes[0].childNodes[3]
+            if(element.tagName == "SELECT"){
+                document.getElementById(element.value).classList.remove(this.hiddenClassValue)
```

Comment:
and in a stimulus controller, don't use `document.getElementById` and `document.getElementsByClassName`;
instead, use this.element and stimulus targets